sncosmo/plotting.py

```diff
@@ -20,10 +20,11 @@
 
 
 def plot_lc(data=None, model=None, bands=None, zp=25., zpsys='ab',
-            pulls=True, xfigsize=None, yfigsize=None, figtext=None,
+            pulls=True, xfigsize=None, yfigsize=None, figtext=None, 
```

Comment:
`fill_percentiles`?

On Tue, Jul 12, 2016 at 10:41 PM, Kyle Barbary <notifications@github.com>
wrote:

> In sncosmo/plotting.py
> <https://github.com/sncosmo/sncosmo/pull/148#discussion_r70566738>:
>
> > @@ -20,10 +20,11 @@
> >
> >
> >  def plot_lc(data=None, model=None, bands=None, zp=25., zpsys='ab',
> > -            pulls=True, xfigsize=None, yfigsize=None, figtext=None,
> > +            pulls=True, xfigsize=None, yfigsize=None, figtext=None,
>
> whitespace, probably causing a pep8 test failure.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sncosmo/sncosmo/pull/148/files/0517c3ba1eeb0fe0007e5f9db3216df001f5f2b7#r70566738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACpC4AMPBiquZbjKdo5hfOdaXVic3yOtks5qVHqQgaJpZM4JK96o>
> .
>



-- 
Danny Goldstein
http://astro.berkeley.edu/~dgold/